docs/Container-Image.md

```diff
@@ -23,4 +23,8 @@ docker run --rm -v "$(pwd)/berlin:/var/opentripplanner" docker.io/opentripplanne
 docker run -it --rm -p 8080:8080 -v "$(pwd)/berlin:/var/opentripplanner" docker.io/opentripplanner/opentripplanner:latest --load --serve
 ```
 
-Now open [http://localhost:8080](http://localhost:8080) to see your running OTP instance.
\ No newline at end of file
+Now open [http://localhost:8080](http://localhost:8080) to see your running OTP instance.
+
+## Additional notes
+Make sure to include the word "gtfs" when naming the gtfs files that you want to use for OTP. Otherwise, graph build will fail.<br/>
```

Comment:
```suggestion
Make sure to include the word "gtfs" when naming the gtfs files that you want to use for OTP. Otherwise, graph build will fail.


```